### PR TITLE
🚧  on EditJobseeker Profile

### PR DESCRIPTION
**Pull request by Nejo12**
* Still under construction.
* few form inputs implemented and tested.
* form submit is also implemented.
* status message: `data and salt arguments required`, which possibly depicts that salt is correctly generated but password is undefined. i.e. verification works and needs a valid logged in user
* need team's clarification on jobseeker registration, or possibly default logged in user to work with.

### CODE REVIEW

File: src/redux/actions/jobseeker.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {
+  REGISTER_JOBSEEKER_DATA_REQUEST,
+  REGISTER_JOBSEEKER_DATA_SUCCESS,
+  REGISTER_JOBSEEKER_DATA_FAIL,
+  Credential,
+} from '../types'
```
Comment (Tsaikkari) [inline src/redux/actions/jobseeker.ts]: I've already made jobseeker register and login.

Comment (Nejo12) [reply src/redux/actions/jobseeker.ts]: Got it. Will change it to UPDATE_JOBSEEKER. Does it fit right?

Comment (Tsaikkari) [reply src/redux/actions/jobseeker.ts]: Yes, you can do the UPDATE_JOBSEEKER_REQUEST etc. Unfortunately, the registerJobseeker doesn't work yet; I think the backend team hasn't removed skills from the register yet.

**Comment by dilippoudel** [comment]
I didn't see the updated code that you have to pull from develop branch. There is the latest code in develop branch since today at 13.00. :) so, please update the latest one and merge with your changes and push it again

**Review by Wyfy0107 — APPROVED** [review]

**Pull request merged**